### DISCUSSION TITLE
I want to know why the cookie-reuse parameter has disappeared in all templates

### DISCUSSION THREAD

**Discussion by ymxlj** [projectdiscovery/nuclei-templates#8777] [Q&A]
### Issue description:

I want to know why the cookie-reuse parameter has disappeared, in all the templates. Is it because the new version of nuclei no longer supports this parameter?

**Comment by ehsandeep** [comment]
@ymxlj since [nuclei v3.0.4 release](https://github.com/projectdiscovery/nuclei/releases/tag/v3.0.4), cookie reuse behavior is enabled as default, so no need to specify it in template and optionally you can disable it - https://docs.projectdiscovery.io/templates/protocols/http/basic-http#session

**Comment by princechaddha** [comment]
Feel free to reopen the discussion if you need any assistance. You can also join our [Discord server](https://discord.com/invite/projectdiscovery), which is a great place to connect with fellow contributors and stay updated on the latest developments.
